hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2339,9 +2342,16 @@ boolean snapshotCleanupSwitch(final boolean on, final boolean synchronous)
    * @param logQueryFilter filter to be used if provided (determines slow / large RPC logs)
    * @return online slowlog response list
    * @throws IOException if a remote or network exception occurs
+   * @deprecated since 2.4.0 and will be removed in 4.0.0.
```

Comment:
As I mentioned in my earlier comments, I recognize it's unfortunate to have an almost immediate deprecation of this but it's better to move to a generic API now then bake in more special case APIs per ring buffer element type. 

So +1 to this.